docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -129,21 +136,31 @@ researchers at Amgen, Inc, and you can read the paper in which it was published
 
 .. image:: MM_graph_time_varying_rates.jpg
 
-.. _allowing_other_complexities:
 
-Allowing other complexities
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Surfacing limitations
+~~~~~~~~~~~~~~~~~~~~~
 
 Another benefit of microsimulations in general is that they can surface complications or limitations that are obscured in non-individual-based models, such as compartmental models.
 
+We can revisit our IV iron intervention microsimulation to illustrate. The graph below depicts individual hemoglobin concentration over time, which as described earlier, is an
+important risk exposure variable in this model. When examining this curve, you may notice minute discrepancies from what you would expect to see in the real world. These are 
```

Comment:
I would include an example of a discrepancy - the biggest one is that there are point in time "jumps" in hemoglobin, whereas in real life a person's hemoglobin moves in a more curved, natural form. It doesn't magically jump up or down the instant you hit 6 weeks postpartum!

Reply:
Yes I wasn't exactly sure what the discrepancy was - I'll add that, thank you Syl!